lib/newman_conway.py

```diff
@@ -1,10 +1,32 @@
-
-
-# Time complexity: ?
-# Space Complexity: ?
 def newman_conway(num):
     """ Returns a list of the Newman Conway numbers for the given value.
         Time Complexity: ?
         Space Complexity: ?
+        P(1) = 1 at 1st index?
```

Comment:
✨ Your implementation works, but what about space and time complexity?

Reply:
Time Complexity: O(n)
 - The while and for loops have a linear time complexity since I am looping through the entire array once and using the append function which has a constant time complexity. Join also has a linear time complexity, so the total time complexity is O(n)

Space Complexity: O(n)
 - I am creating a data structure that grows linearly